tests/scripts/remote_monitoring_tests/helpers/json_validate.py

```diff
@@ -0,0 +1,128 @@
+"""
+Copyright (c) 2022, 2022 Red Hat, IBM Corporation and others.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import jsonschema
+from jsonschema import validate, draft7_format_checker
+
+DIRECTIONS_SUPPORTED = ("maximize", "minimize")
+
+DIRECTION_NOT_SUPPORTED = "Direction not supported!"
+JSON_NULL_VALUES = ("is not of type 'string'", "is not of type 'integer'", "is not of type 'number'")
+VALUE_MISSING = " cannot be empty or null!"
+
+exp_input_schema = {
+    "type": "array",
+    "items": {
+        "type": "object",
+        "properties": {
+            "name": {"type": "string"},
+            "namespace": {"type": "string"},
+            "slo": {
```

Comment:
What about `performanceProfile`? It can be either `slo` or `performanceProfile` that can be specified. Infact in the remote_monitoring case, it will always be `performanceProfile`

Reply:
I forgot to remove this file, will remove it. I 'm not using this for the tests, had added it only for the demo as the json validation will need to be done in the source code.